opentelemetry-otlp/tests/integration_test/tests/logs.rs

```diff
@@ -34,14 +38,23 @@ fn init_logs() -> Result<sdklogs::LoggerProvider, LogError> {
         .build())
 }
 
-pub async fn logs() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+#[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+pub async fn test_logs() -> Result<()> {
```

Comment:
These are regular tests now, they just need to make sure the container is running with the first line

Reply:
can you add code comment or a md file on how one can run this locally? like listing the pre-requisites (only docker, I guess?)